src/services/user.service.js

```diff
@@ -0,0 +1,36 @@
+const { User } = require('../models/User.model');
+const { generateRandomId } = require('../Random');
+
+const getAll = async () => {
+  return User.findAll();
+};
+
+const getById = async (id) => {
+  return User.findByPk(+id);
+};
+
+const updateUser = async ({ id, name }) => {
+  await User.update({ name }, { where: { id } });
+};
+
+const createUser = async (name) => {
+  const id = generateRandomId();
+
+  return User.create({ id, name });
```

Comment:
Consider adding validation for the 'name' parameter to ensure it meets any necessary criteria (e.g., not empty, within character limits) before attempting to create a user.